Users/Install.rst

```diff
@@ -83,10 +83,82 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is the most user friendly method of managing virtual environment and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command.
+
+::
+
+    $ pip install pipenv
+
+Now we create a python 3 environment for our project, move to your project
+directory and run the following command.
+
+::
+
+    $ pipenv --three
+
+The above command will create virtual environment isolated from your system
+global installation.
+
+    On **Unix based** systems (OS X and Linux), you can find the user base
+    binary directory for your pipenv by running
+
+    ::
+
+        $ python -m site --user-base
+
+    Result of the above command will be something similar to
+    ``/home/<Username>/.local``.
+    And on **Windows**, run the following command
+
+    ::
+
+        $ py -m site --user-site
+
+    Result of the above command will be something similar to
+    ``C:\Users\<Username>\AppData\Roaming\Python36\site-packages``.
+    You'll have to add this result after replacing ``site-packages``
+    with ``Scripts`` i.e.
+    ``C:\Users\<Username>\AppData\Roaming\Python36\Scripts``
+    to PATH from Control Panel. This is only when you don't have this PATH
+    already set.
+
+To install coala, run the following command
+
+::
+
+    $ pipenv install coala-bears
+
+Great, you have now setup a pipenv environment for your project directory and
+installed coala using it.
+You can now activate the pipenv by running the following
+
+::
+
+    $ pipenv shell
+
+Your virtual environment (virtualenv) is now active. You'll notice the change
+before ``$`` sign on the command-line which will have your project directory
+name and some alpha-numeric value for uniqueness.
+You can use coala by running ``coala`` command and exit the virtualenv
+environment by entering ``exit`` at any time.
+
+You can read more about `pipenv <https://docs.pipenv.org>`_ from it's
+documentation.
+
 Installing inside a virtualenv
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Virtualenv is probably what you want to use during development.
+If you feel the need to use Virtualenv seperately you can follow the steps
```

Comment:
Lowercase virtualenv

Reply:
sure. in that case, I'll have to add it inside code cases like `virtualenv`.